File: Assignment_15/Program2.c
#include <stdio.h>

void Pattern(int iRow, int iCol)
{
    int iCnt1 = 0, iCnt2 = 0;
    for(iCnt1 = 1; iCnt1 <= iRow ; iCnt1++)
    {
        for(iCnt2 = iCol; iCnt2 >= 1; iCnt2--)
        {
           if(iCnt2 >= iCnt1)
           {
            printf("*\t");
           }
           else
           {
            printf("#\t");
           }
        }
            printf("\n");

    }

}
int main()
{
    
   int iValue1 = 0, iValue2 = 0;

   printf("Enetr number Rows : \n");
   scanf("%d",&iValue1);

   printf("Enetr number Columns : \n");
   scanf("%d",&iValue2);

    
   Pattern(iValue1, iValue2);



    return 0;
}

File: Assignment_50/Program2.java
import java.util.*;

class Pattern
{
    private int iRow;
    private int iCol;


    Pattern(int iRow,int iCol)
    {
        this.iRow = iRow;
        this.iCol = iCol;
    }

    public void Display()
    {
        int i = 0,j = 0;

        for(i = iRow-1; i >= 0; i--)
        {
            for(j = 0; j < iCol;j++)
            {
                if(i == j)
                {
                    System.out.print("#\t");
                }
                else if(i > j)
                {
                    System.out.print("*\t");
                }
                else
                {
                    System.out.print("@\t");
                }
            }
            System.out.println( );
        }
    }
}

class Program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows  : ");
        int iRow = sobj.nextInt();

        System.out.println("Enter number Columns : ");
        int iCol = sobj.nextInt();

        Pattern pobj = new Pattern(iRow,iCol);

        pobj.Display();

    }
    
}